submissions/dafen173/js-dom/script.js

```diff
@@ -0,0 +1,171 @@
+
+// Menu button
+const menubtn = document.getElementById('menubtn');
+const dropchild = document.getElementById('dropchild');
+menubtn.addEventListener("click", function(){
```

Comment:
Dropdown doesn't work. You add/remove class `.open` but nothing happening.

Reply:
I assigned this class only for
@media screen and (max-width: 750px)

so okay?

Reply:
?

Reply:
Yes, it's ok.
But mobile viewport starting from 767px and going down up to 320px. So better use this case for @media screen and (max-width: 767px).

Reply:
understood, accepted
)

Reply:
done

https://github.com/dafen173/kottans_frontend/blob/master/task_js_dom/style.css

ok?